README.md

```diff
@@ -107,6 +107,70 @@ end
 
 _Note: `drop_split` (a.k.a. `finish_split`) does not physically delete split data from mixpanel or Test Track's database._
 
+### Generating Split Migrations
+
+Split configuration changes can be generated using Rails generators that are included with the TestTrack Rails client.
+
+```
+rails generate test_track:migration add_name_of_split
```

Comment:
`add_[name_of_feature]` might be a good way to talk about this instead of `name_of_split` if we're going to be suffixing it with either `_experiment` or `_enabled` if we're given a type?

Also ActiveRecord migrations can infer type from the name of the migration - maybe you can get the same effect as `-t drop` by calling with `drop_[name_of_feature]`, or the same effect as `-t experiment` by calling with `add_[name_of_feature]_experiment`.